### PR TITLE
Todays maintenance releases

### PR DESCRIPTION
**Pull request by MorrisJobke**
Basically this:

```
  "dueDates": {
    "Nextcloud 16.0.11": "2020-06-04",
    "Nextcloud 16.0.12": "2020-07-16",
    "Nextcloud 17.0.7": "2020-06-04",
    "Nextcloud 17.0.8": "2020-07-16",
    "Nextcloud 18.0.5": "2020-06-04",
    "Nextcloud 18.0.6": "2020-07-16"
  },
  "closeMilestones": [
    "Nextcloud 16.0.10",
    "Nextcloud 17.0.6",
    "Nextcloud 16.0.4"
  ],
  "addMilestones": [
    "Nextcloud 16.0.11",
    "Nextcloud 16.0.12",
    "Nextcloud 17.0.7",
    "Nextcloud 17.0.8",
    "Nextcloud 18.0.5",
    "Nextcloud 18.0.6",
    "Nextcloud 19"
  ],
```

### CODE REVIEW

File: milestoneupdater/config.json [hunk 1]
```diff
@@ -33,26 +33,26 @@
     "viewer"
   ],
   "dueDates": {
-    "Nextcloud 15.0.12": "2019-09-26",
-    "Nextcloud 15.0.13": "2019-11-07",
-    "Nextcloud 16.0.5": "2019-09-26",
-    "Nextcloud 16.0.6": "2019-11-07"
+    "Nextcloud 16.0.11": "2020-06-04",
+    "Nextcloud 16.0.12": "2020-07-16",
```
Comment (rullzer) [inline milestoneupdater/config.json:37]: 16.0.11 is the latest as far as I know

Comment (MorrisJobke) [reply milestoneupdater/config.json:37]: latest or last?

Comment (MorrisJobke) [reply milestoneupdater/config.json]: Done.

File: milestoneupdater/config.json [hunk 2]
```diff
@@ -33,26 +33,24 @@
...
+    "Nextcloud 18.0.6": "2020-07-16"
   },
   "closeMilestones": [
-    "Nextcloud 14.0.14",
-    "Nextcloud 14.0.15",
-    "Nextcloud 14.0.16",
-    "Nextcloud 15.0.11",
+    "Nextcloud 16.0.10",
+    "Nextcloud 17.0.6",
     "Nextcloud 16.0.4"
```
Comment (rullzer) [inline milestoneupdater/config.json:45]: 18.0.4

Comment (MorrisJobke) [reply milestoneupdater/config.json]: Done.

**Pull request merged**